activerecord/test/cases/callbacks_test.rb

```diff
@@ -78,6 +78,42 @@ def cancel
     end
 end
 
+class RelatedProject < ActiveRecord::Base
+  self.table_name = 'projects'
+end
+
+class RelatedDeveloper < ActiveRecord::Base
+  self.table_name = 'developers'
+end
+
+class RelatedDeveloperProject < ActiveRecord::Base
+  self.table_name = 'developers_projects'
+
+  belongs_to :developer, :class => RelatedProject, :foreign_key => 'developer_id'
```

Comment:
Thanks. That's great to know. Indeed, I wasn't sure whether the new hash syntax would be allowed. Forgot Ruby 1.9 support has been dropped from newer Rails releases. I'll use the new hash syntax in my code for my upcoming revisions on this Pull Request.